model/registry/gen-ai.yaml

```diff
@@ -0,0 +1,67 @@
+groups:
+  - id: registry.llm
+    prefix: gen_ai.llm
+    type: attribute_group
+    brief: >
+      This document defines the attributes used to describe telemetry in the context of LLM (Large Language Models) requests and responses.
+    attributes:
+      - id: system
+        type: string
```

Comment:
we should convert it into the enum and add openai as the only member (check out `http.request.method` or any other existing enum attribute as an example)

Reply:
If an enum, this would have weekly additions and grow very large?

Reply:
Could you clarify? My assumption was that we have only one candidate to put here initially - `openai`. We don't need to define constants for all the possible GenAI vendors in the world, we really want to include those who
- need special semantics and have an extension
- have dedicated instrumentation library and are willing to come and add a constant to the spec.

If list grows too much (DB one has ~50 names), we'll figure out what to do. We can always move it to a separate file and also can implement enum extension mechanism. 

Let me know if I missed something, I'm open to any suggestions.

Reply:
I agree @lmolkova